### DISCUSSION TITLE
organize creates a different folder name than requested

### DISCUSSION THREAD

**Discussion by alwayserik** [tfeldmann/organize#136] [Q&A]
rules:
  - folders: '/run/media/erik/Filesystem1/temp'
    filters:
      - created:
          days: 1
          mode: older
    actions:
      - echo: 'Was created over a day ago'
      - copy: '/run/media/erik/Filesystem3/temp'

when I run sim:

[erik@slater02 ~]$ organize sim
True
SIMULATION
Folder /run/media/erik/Filesystem1/temp:
  File ntfsmwts1.jpg:
    - [Echo] Was created over a day ago
    - [Copy] Copy to "/run/media/erik/Filesystem3/temp 2"
  File ntfsmwts2.jpg:
    - [Echo] Was created over a day ago
    - [Copy] Copy to "/run/media/erik/Filesystem3/temp 2"
  File ntfsmwts3.jpg:
    - [Echo] Was created over a day ago
    - [Copy] Copy to "/run/media/erik/Filesystem3/temp 2"

the question is, why does it copy it to "temp 2" rather than "temp" as requested? Can this be fixed or specified?

**Comment by wintuxx** [comment]
Hello, 
you must end the folder with /

folders: '/run/media/erik/Filesystem1/temp/'

copy: '/run/media/erik/Filesystem3/temp/'

with your settings organize will create a file named temp

**Reply by alwayserik** [reply]
Thanks, that worked!!

By the way, another question for ya. If the destination runs out of space, will it copy? How does it respond?

**Reply by tfeldmann** [reply]
If there's no space it will not copy and error out
